apps/server/src/modules/files/job/delete-files.console.spec.ts

```diff
@@ -1,6 +1,8 @@
 import { createMock } from '@golevelup/ts-jest';
 import { Test, TestingModule } from '@nestjs/testing';
+
```

Comment:
???

Reply:
I like to visually separate the imports into three parts:

1. the external, 3rd-party libraries imports
2. the project-scoped, imports, like the logger library shared between many modules
3. the "local", module-scoped imports, like the local files imports.

I think it creates a better separation and I think it's quicker to understand what's the real source of a specific imports.

Reply:
We use auto formating.

Reply:
@SevenWaysDP ok, I'll modify all the imports then to delete all the empty lines and just put all the imports line-by-line.